modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -292,7 +312,23 @@
   [_]
   :sunday)
 
-
 (defmethod driver/notify-database-updated :bigquery-cloud-sdk
   [_ database]
   (bigquery.common/populate-project-id-from-credentials! database))
+
+(defmethod driver/normalize-db-details :bigquery-cloud-sdk
+  [_ database]
+  (if-let [dataset-id (get-in database [:details :dataset-id])]
+    (if-not (str/blank? dataset-id)
```

Comment:
I don't see this getting rid of `[:details :dataset-id]` so won't this normalization run every time the normalize-db-details is called? Should this normalize update the condition that indicates it needs to normalize?

Reply:
There is an `m/dissoc-in` below that does it